metro.config.ci.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  transformer: {
+    getTransformOptions: async () => ({
+      transform: {
+        experimentalImportSupport: false,
+        inlineRequires: false
```

Comment:
Are you sure this is already used? https://facebook.github.io/react-native/blog/2019/03/12/releasing-react-native-059#faster-app-launches-with-inline-requires

Not sure which side-effects this has

Reply:
Ahh, no then it's not used yet. I'll remove it...